mtags/src/main/scala/scala/meta/internal/pc/HoverProvider.scala

```diff
@@ -32,7 +33,8 @@ class HoverProvider(val compiler: MetalsGlobal, params: OffsetParams) {
           val expanded = expandRangeToEnclosingApply(pos)
           if (expanded != null &&
             expanded.tpe != null &&
-            tree.symbol != null) {
+            tree.symbol != null &&
+            expanded.symbol != null) {
```

Comment:
🙈

Reply:
That's the only thing we didn't check haha